packages/simcore-sdk/src/simcore_sdk/node_ports/filemanager.py

```diff
@@ -28,7 +28,11 @@ class ClientSessionContextManager:
     # See https://github.com/ITISFoundation/osparc-simcore/issues/1098
     #
     def __init__(self, session=None):
-        self.active_session = session or ClientSession()
+        # We are interested in fast connections, if a connection is established
+        # there is no timeout for file download operations
+        self.active_session = session or ClientSession(
+            timeout=ClientTimeout(total=None, connect=5, sock_connect=5)

```

Comment:
this library has access to models-library. you could get the variables from there. I would suggest using:
class ClientRequestSettings(BaseSettings):
  io_request_total_timeout = None
  io_request_connect...

Reply:
We can put them there, no issue issue here. let's have all these on ENV vars to keep timeouts customisable on the fly.